apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/get/GetAccountLessors.java

```diff
@@ -57,13 +57,16 @@ public JSONStreamAware processRequest(HttpServletRequest req) throws AplExceptio
         response.put("height", height);
         JSONArray lessorsJSON = new JSONArray();
 
-        try (DbIterator<Account> lessors = account.getLessors(height)) {
+        try (DbIterator<Account> lessors = lookupAccountService().getLessorsIterator(account,height)) {
             if (lessors.hasNext()) {
                 while (lessors.hasNext()) {
                     Account lessor = lessors.next();
                     JSONObject lessorJSON = new JSONObject();
                     JSONData.putAccount(lessorJSON, "lessor", lessor.getId());
-                    lessorJSON.put("guaranteedBalanceATM", String.valueOf(lessor.getGuaranteedBalanceATM( CDI.current().select(BlockchainConfig.class).get().getGuaranteedBalanceConfirmations(), height)));
+                    lessorJSON.put("guaranteedBalanceATM"
+                            , String.valueOf(lookupAccountService().getGuaranteedBalanceATM(lessor,
+                                    CDI.current().select(BlockchainConfig.class).get().getGuaranteedBalanceConfirmations()
```

Comment:
I would suggest to replace
`CDI.current().select(BlockchainConfig.class).get()`
by lookupXXX() created inside AbstractAPIRequestHandler as that is done for another CDI components.

Reply:
This class is being refactored now and marked as Deprecated, need to be removed after that.